ament_tools/build_types/cmake.py

```diff
@@ -70,6 +72,10 @@ def prepare_arguments(self, parser):
             help="Arbitrary arguments which are passed to all CTest invocations. "
                  "The option is only used by the 'test*' verbs. "
                  "Argument collection can be terminated with '--'.")
+        parser.add_argument(
+            '--ninja',
+            action='store_true',
+            help="Invoke 'cmake' with -G Ninja and call ninja instead of make.")
```

Comment:
Can you please name the option `--use-ninja` (to also have a similar `--use-nmake` option in the future) to be the same as in existing ROS 1 tools (see https://github.com/ros/catkin/blob/3519c73394d2121b85ddbee488e4c44d4d942118/bin/catkin_make#L267).